### PR TITLE
Use class name resolution via ::class

### PR DESCRIPTION
**Pull request by billmn**

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
No point.

File: app/Console/Kernel.php [hunk 1]
```diff
@@ -22,7 +22,7 @@ class Kernel extends ConsoleKernel
      * @var array
      */
     protected $commands = [
-        'CachetHQ\Cachet\Console\Commands\DemoSeederCommand',
+        \CachetHQ\Cachet\Console\Commands\DemoSeederCommand::class,
```
Comment (GrahamCampbell) [inline app/Console/Kernel.php]: maybe this one can stay, but it would make sense to import it

**Comment by billmn** [comment]
The sense of this PR is to use a single "method" to specify class names.

Some parts of Cachet already use this (like models) and can also be useful to keep configuration files updated with Laravel.

**Comment by GrahamCampbell** [comment]
It's not consistent though. We NEVER reference classes using a leading slash. We always import them, and we're not going to import like 50 classes to use ::class.

**Comment by GrahamCampbell** [comment]
This is only useful when referencing our own code at best.

File: config/app.php [hunk 2]
```diff
@@ -122,56 +122,56 @@
...
-        'Illuminate\Pagination\PaginationServiceProvider',
-        'Illuminate\Pipeline\PipelineServiceProvider',
-        'Illuminate\Queue\QueueServiceProvider',
-        'Illuminate\Redis\RedisServiceProvider',
-        'Illuminate\Auth\Passwords\PasswordResetServiceProvider',
-        'Illuminate\Session\SessionServiceProvider',
-        'Illuminate\Translation\TranslationServiceProvider',
-        'Illuminate\Validation\ValidationServiceProvider',
-        'Illuminate\View\ViewServiceProvider',
+        Illuminate\Foundation\Providers\ArtisanServiceProvider::class,
```
Comment (GrahamCampbell) [inline config/app.php]: we're never going to do this in config files - i think it's a stupid idea

**Comment by billmn** [comment]
Ok make sense, thank you

File: app/Console/Kernel.php (hunk 1 above)
Comment (GrahamCampbell) [inline app/Console/Kernel.php]: https://github.com/cachethq/Cachet/commit/814fcd844c30aa2bfc1f3bb96812a3f678a49450

File: config/app.php (hunk 2 above)
Comment (billmn) [inline config/app.php]: ok I got it

**Pull request closed without merging**